UE4Project/FSOnline.uproject

```diff
@@ -1,6 +1,6 @@
 {
 	"FileVersion": 3,
-	"EngineAssociation": "4.25",
+	"EngineAssociation": "4.24",
```

Comment:
Is there a reason for this change? Or is it an accident?

Reply:
This is an accident. Not sure how that got changed!